pabi_procurement/models/purchase_request.py

```diff
@@ -80,10 +80,12 @@ class PurchaseRequest(models.Model):
         'res.currency',
         string='Currency',
         required=True,
+        default=140,
```

Comment:
Do not hard code default currency.
You can use Purchase Request's Company's Currency, i.e.,
`
default=lambda self: self.company_id.currency_id.id,
`
You can use Purchase Request's Company's Currency,